libs/ssl-config/src/main/java/org/opensearch/common/ssl/DefaultJdkTrustConfig.java

```diff
@@ -65,7 +65,7 @@ final class DefaultJdkTrustConfig implements SslTrustConfig {
      * Create a trust config that uses supplied {@link BiFunction} to determine the TrustStore type, and the relevant password.
      */
     DefaultJdkTrustConfig(BiFunction<String, String, String> systemProperties) {
-        this(systemProperties, isPkcs11Truststore(systemProperties) ? getSystemTrustStorePassword(systemProperties) : null);
+        this(systemProperties, getSystemTrustStorePassword(systemProperties));
```

Comment:
What is the expected completion of `KeyStoreUtil.readKeyStore(path, "BCFKS", trustStorePassword);` meant to be? It looks like it will not populate this field with null even under the successful read of the Bcfks trustore. Is that appropriate?